scripts/yaml_check.py

```diff
@@ -140,16 +144,23 @@ class ThousandPlusTheorem:
     try:
         _thm = ThousandPlusTheorem(index, **entry)
     except TypeError as e:
-        print(f"error: entry for theorem {index} is invalid: {e}")
+        print(f"error: entry for theorem {index} is invalid: {e}", file=sys.stderr)
         errors += 1
     # Also verify that the |decl| and |decls| fields are not *both* provided.
     if _thm.decl and _thm.decls:
         print(
-            f"warning: entry for theorem {index} has both a decl and a decls field; "
+            f"error: entry for theorem {index} has both a decl and a decls field; "
             "please only provide one of these",
             file=sys.stderr,
         )
         errors += 1
+    elif _thm.statement and (_thm.decl or _thm.decls):
+        print(
+            f"error: entry for theorem {index} has both a statement and a decl(s) field: "
+            "the latter is superfluous; please remove it",
```

Comment:
If `decl(s)` is present, then `statement` is the one that is superfluous, right?
```suggestion
            "the former is superfluous; please remove it",
```